src/main/java/com/bitdecay/ludum/dare/control/GameControls.java

```diff
@@ -0,0 +1,11 @@
+package com.bitdecay.ludum.dare.control;
+
+import com.badlogic.gdx.Input;
+
+public class GameControls {
+    public static GameControl Left = new GameControl("[A] or [LEFT]", Input.Keys.A, Input.Keys.LEFT);
```

Comment:
as a total side note -- in playing around with Jump (which now shows all the keys and what they do) I learned that you can do `Input.Keys.toString(Input.Keys.LEFT)` and it'll dump out a human readable string for the key name.